conda/recipes/cudf_kafka/meta.yaml

```diff
@@ -34,7 +35,7 @@ requirements:
   run:
     - python {{ py_version }}
     - libcudf_kafka {{ version }}
-    - python-confluent-kafka >=1.7.0,<1.8.0a0=py{{ py_version.replace('.', '') }}*
+    - python-confluent-kafka >=1.7.0,<1.8.0a0=py{{ py_version_numeric }}*
```

Comment:
@jakirkham, do you know if the `=py{{ py_version_numeric }}*` suffix extraneous here? `conda` should solve for the correct Python version without it, right?

Reply:
Merging this PR for now. We can always remove this in the future if needed.